src/instructlab/model/evaluate.py

```diff
@@ -691,7 +695,21 @@ def evaluate(
             display_error_rate((error_rate + base_error_rate) / 2)
 
         elif benchmark == Benchmark.MMLU:
-            overall_score, individual_scores = evaluator.run()
+            server = None
+            try:
+                server, api_base = launch_server(
+                    ctx,
+                    model,
+                    model,
+                    None,
```

Comment:
Why is `max_workers` set to None for MMLU?

Reply:
max_workers is only for the mt_bench variants

Reply:
Ok but given that MMLU now uses the same serving backend and that both backend supports max_workers AFAIK I don't understand why max_workers wouldn't be applicable. Could you please clarify for me? Thanks!

Reply:
max_workers isn't used by the serving, it's used by the evaluation logic itself.  Specifically, mt_bench uses a pool of workers to run through the evaluation questions and max_workers is the tuning param for that pool.